public/index.html

```diff
@@ -127,6 +116,16 @@ <h5 id="gameInstructions"></h5>
             <form class="form" id="gameForm" action=""></form>
         </div>
         <div class="remaining">
+            <div class="rooms">
+                <div class="half">
+                    <h4 id="room" align="center">Room: Lobby</h4>
+                    <ul id="roomPlayers" align="center" width="80%"></ul>
+                </div>
+                <div class="half">
+                    <h4 id="gameID" align="center">Game: not in a game</h4>
+                    <ul id="gamePlayers" align="center" width="80%"></ul>
+                </div>
+            </div>
```

Comment:
Any special reason for this change?

I think I preferred when the table was at the top.

Reply:
Ok, I'll move them back to the top.